src/main/java/pl/javastart/dictionary/LinguController.java

```diff
@@ -29,7 +29,7 @@ public LinguController(EntryRepository entryRepository, FileService fileService,
     }
 
     void mainLoop() {
-        outputWriter.println("Witaj w aplikacji LinguApp");
```

Comment:
Tutaj napisze uwage to konstruktora wyzej - ogolnie zgodnie z dokumentacja, od springa 4.3 jezeli jest pojedynczy kontruktor to te adnotacje @Autowired mozna pominąc. Jednak część osób zostawia dla czytelnosci kodu (ja w sumie zostawiam) - to tak informacyjnie.

Reply:
zrobione